validation/input/input_1/input_script_1.py

```diff
@@ -29,9 +29,20 @@ def test_progress():
         time.sleep(0.001)
 
 
-def test_cwd():
-    assert (Path(os.environ["INPUT_FOLDER"]) / "input_script_1.py").exists()
-    assert (Path(os.environ["INPUT_FOLDER"]) / "requirements.txt").exists()
+def test_inputs():
+    assert (Path(os.environ["INPUT_1"]) / "input_script_1.py").exists()
+    assert (Path(os.environ["INPUT_1"]) / "requirements.txt").exists()
+
+
+def test_outputs():
+    """write a file in each output"""
+    for output_folder_env in [*(f"OUTPUT_{i}" for i in range(1, 5))]:
+        folder = Path(os.environ[output_folder_env])
+        assert folder.exists()
+        for number in range(1, 3):
```

Comment:
MINOR: your intention here is to test that you have access writes to this folder, right?
IMO that intention can be easily expressed as

```python
for env_var in ( ... ):
    folder = Path(env_var)
    assert folder.exists()
    (folder / "can-write-file").touch()

```

Reply:
Aha ... now I see you use it to produce outputs equivalent to ``validation/output`` ... to pass the other test.
Kind of recursive testing here :-)

SUGGESTION: could you please add a note to clarify that this reproduces validateion_outputs ... to avoid messing up with the test implementation (as I was trying above). thx!

Reply:
ahah actually I generated the validation code by running ```make up```
then copied the output to the validation/output